spack_environments/llnl_lc/externals-linux-rhel7-broadwell.sh

```diff
@@ -8,9 +10,12 @@
       version: []
       paths: {}
       modules: {}
-      compiler: [gcc@7.3.0 arch=linux-rhel7-broadwell]
+EOF
+)
 
+EXTERNAL_PACKAGES=$(cat <<EOF
     cmake::
+      buildable: True
       variants: ~openssl ~ncurses
       version: [3.12.1]
```

Comment:
This should be bumped to at least 3.15. DiHydrogen, e.g., will depend on a more recent CMake.

Reply:
Grr.  LC doesn’t have anything past 3.14.

Reply:
I think that we will bump this in another PR.